qiskit/transpiler/passes/layout/vf2_layout.py

```diff
@@ -196,21 +211,36 @@ def _score_layout(self, layout):
         on the chosen qubits. If BackendProperties are not available it uses the coupling map degree
         to weight against higher connectivity qubits."""
         bits = layout.get_physical_bits()
-        score = 0
-        if self.properties is None:
-            # Sum qubit degree for each qubit in chosen layout as really rough estimate of error
+        score = None
+        if self.target is not None:
+            if "measure" in self.target:
+                score = 0
+                for bit in bits:
+                    props = self.target["measure"].get((bit,))
+                    if props is None or props.error is None:
+                        score += (
+                            self.coupling_map.graph.out_degree(bit)
+                            + self.coupling_map.graph.in_degree(bit)
+                        ) / len(self.coupling_map.graph)
+                    else:
+                        score += props.error
+        if score is None:
```

Comment:
I think this is better, because it is clear how `target` supersedes `properties`. And removes the need to distinguish `score is None` from `score == 0` 
```suggestion
        else:
```